autoscheduler/autoscheduler/settings.py

```diff
@@ -70,17 +71,28 @@
 WSGI_APPLICATION = 'autoscheduler.wsgi.application'
 
 
+# read postgres username/password from config/postgres-information.json
+infoFile = os.path.dirname(__file__) + "/config/postgres-information.json"
+try:
+    info = json.load(open(infoFile, "r"))
+    user = info.get("user")
+    password = info.get("password")
+    if user == "":
+        print(f"Please configure {os.path.abspath(infoFile)} to include your postgres username and password.")
+        exit()
+except FileNotFoundError as error:
+    print(error)
+    exit()
 # Database
 # https://docs.djangoproject.com/en/2.2/ref/settings/#databases
-
 DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.postgresql',
         'NAME': 'dbautoscheduler',
         'HOST': 'localhost',
         'PORT': '5432',
-        'USER': '',
-        'PASSWORD': '',
+        'USER': user,
+        'PASSWORD': password,
```

Comment:
Config for username and password can be done in a separate file, just call functions